CONTRIBUTING.md

```diff
@@ -106,7 +106,7 @@ $ git branch -D <branch-name>
 
 ## Deleting the branch in remote repository
 $ git push origin --delete <branch-name>
-```
+
```

Comment:
Why did you edit the `CONTRIBUTING.md` ? We prohibit that. There's a minor change in it, but still.

Reply:
How can i do that? Kindly help.